docs/sigs/Certification.md

```diff
@@ -15,36 +15,49 @@ We use [mattermost SIG/Certification](https://chat.almalinux.org/almalinux/chann
+    -   This acts as the primary way for the community to engage with each other (including the SIG members) for work on certifications or the certification suite. The most collaborative place for the SIG is this chat room. This room is also bridged to matrix on [#sig-certification:almalinux.im](https://app.almalinux.im/#/room/#sig-certification:almalinux.im)
+-   Private certification mailing list
+    -   The ([certification-sig\@lists.almalinux.org](mailto:certification-sig@lists.almalinux.org?subject=updateme)) mailing list acts as a private place for the Certification SIG to communicate with IHVs and ISVs. If you require your interaction with the SIG to be private (for example, if you are working with the SIG under NDA), you may reach out here.
 
 ### Help wanted
 
-* Review of the AlmaLinux OS Foundation Hardware Certification Program document to ensure its clear and concise.
-* Contributions towards expanding the AlmaLinux Hardware Certification Suite to represent a meaningful range of hardware and Linux hardware driver tests that an end user would expect a certification to cover.
-* There is work in progress happening where an Independent Software Vendor can maintain a set of tests for their software with obligations as an addition part of this suite. Once this is defined more, vendor participation will be encouraged. Let us know if this is you so you can be advised when we are ready.
-* Eventually users will be welcome to run the AlmaLinux Hardware Certification Suite on the hardware they have access to and submit these results.
+<u>Contribute to testing: "Community Validated"</u>
+
+In addition to the certification processes with IHVs managed by the AlmaLinux OS Foundation (ALOSF), we encourage community involvement in hardware testing. Individuals with access to hardware that we have yet to certify are encouraged and invited to participate. This not only amplifies the scope of our hardware compatibility and validates the strength of the certification suite, but also shows IHVs the types and scope of the AlmaLinux user base.
+
+Hardware that has been certified without input from the IHV will be described on the AlmaLinux website with a "Community Certified" status. This serves as a first confirmation of compatibility until a relationship with the IHV can be confirmed, and provides users with confidence in hardware compatibility.
+
+Once the certification suite has been run, results should be shared through a pull request to the [Certifications](https://github.com/AlmaLinux/certification-test-results) repo.
+
+More ways to contribute:
+
+* Contributions towards expanding the [AlmaLinux Hardware Certification Suite](https://github.com/AlmaLinux/Hardware-Certification-Suite) to represent a meaningful range of hardware and Linux hardware driver tests that an end user would expect a certification to cover.
+* Help spin up the software certification program.
+* Validate results submitted and listed on the website already
```

Comment:
maybe change that to Re-validate or "provide updates to"? i.e. to promote people updating existing results e.g. for newer alma versions, different kernels, updated hardware revisions etc?